.github/workflows/ci.yml

```diff
@@ -90,37 +75,53 @@ jobs:
       env:
         COLOR: 'yes'
       run: |
-        make test-only
+        make test
         python -m coverage xml
```

Comment:
@Dreamsorcerer this is a sneaky task: if the coverage invocation succeeds, then the entire step will be green regardless of whether the make command failed. This is why I usually insist on either separate steps, or at least adding `set -Eeuo pipefail` at the top of the step script.